tests/shared/src/test/scala/org/http4s/UriSuite.scala

```diff
@@ -1071,10 +1071,6 @@ class UriSuite extends Http4sSuite {
     assertEquals(updated.renderString, "/")
   }
 
-  test("Uri.renderString should Encode special chars in the query") {
-    val u = Uri(path = Uri.Path.Root).withQueryParam("foo", " !$&'()*+,;=:/?@~")
-    assertEquals(u.renderString, "/?foo=%20%21%24%26%27%28%29%2A%2B%2C%3B%3D%3A/?%40~")
-  }
```

Comment:
Sorry, why did we remove this test? Is it asserting some wrong behavior?

Reply:
Yes, correct. It was good for the old QueryNoEncode characters, but for the new ones, this test case is no longer needed.